datafn/python/tests/test_mutation_enhanced.py

```
@@ -0,0 +1,441 @@
+            "operation": "insert",
+            "id": "task_hack",
+            "record": {"__proto__": {"isAdmin": True}, "title": "hack"},
+        })
+        assert res["ok"] is False
+        assert res["error"]["code"] == "DFQL_INVALID"
+        assert "Prototype pollution" in res["error"]["message"]
+        assert "__proto__" in res["error"]["message"]
+
+    @pytest.mark.asyncio
+    async def test_nested_proto_key_rejected(self):
+        handler, db = await _setup()
```

Ruff (0.15.7)

[warning] Unpacked variable `db` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)